editors/code/src/util.ts

```diff
@@ -0,0 +1,18 @@
+let ENABLED: boolean = false;
+
+export const log = {
+    debug(message?: any, ...optionalParams: any[]): void {
+        if (!ENABLED) return;
+        // eslint-disable-next-line no-console
+        console.log(message, ...optionalParams);
+    },
+    error(message?: any, ...optionalParams: any[]): void {
+        if (!ENABLED) return;
+        debugger;
+        // eslint-disable-next-line no-console
+        console.error(message, ...optionalParams);
+    },
+    setEnabled(yes: boolean): void {
```

Comment:
I'd prefer the following (found it the best for naming booleans), though it's just a nit )
```suggestion
    setEnabled(shouldBeEnabled: boolean): void {
```

Reply:
I like `yes` more. This is a convention I've recently discovered when reading ripgrep source code, and I'd love to use it for all my future code because it is short, unambigious, and doesn't require thinking about it.